### DISCUSSION TITLE
[如何提问] 寻找普通组件的简单思路

### DISCUSSION THREAD

**Discussion by ly525** [liuyanshi/gesang-fe-something#654] [团队管理]
问：有没有前端 思维导图 的组件？

答：
先说结论：
https://github.com/anteriovieira/vue-mindmap

我的一些建议：
1. 查一下思维导图的翻译：mind map
2. 查找：  
  2.1 google：vue/react mind map
  2.2 awesome-vue 搜索
  2.3 mind map made with vue/react
  2.4 github 搜索：mind map，然后按照 language 进行过滤

### 我推荐的提问方式：

我想找一个思维导图组件，希望满足如下条件：
1. 技术栈是 Vue/React/Angular/D3/Canvas 中的哪一种

2. 我想要的效果是什么？有无demo/竞品可以参考（有图有真相，说不定不是思维导图，而是流程图之类）

3. 我自己找了一些，如下：

4. 但不能满足需求，还有一些痛点：痛点A、痛点B等等，希望大家可以帮忙找下能解决这些痛点的组件（表明你已经做了努力，说不定也有这在做这个的，也可以交流）

**Comment by chenyong1991** [comment]
多谢给出的建议
